dashboard/app/models/pd/application/teacher1819_application.rb

```diff
@@ -181,6 +181,7 @@ def enroll_user
       OTHER_PLEASE_LIST
     ]
 
+    NOT_TEACHING_NEXT_YEAR = "I'm not teaching next year (please explain):"
     NOT_TEACHING_THIS_YEAR = "I'm not teaching this year (please explain):"
```

Comment:
super nit: reorder so this year comes before next year (and it'll be consistent with the options below)